### ISSUE TITLE
Insufficient colour contrast for links in final quiz question of Module 13

### ISSUE THREAD

**Issue by python33r** [theodi/ODI-eLearning#488]
In the 'Taylor Swift' question, the answer boxes all look blank. Mousing over them makes the background a very pale grey and it then just about possible to see the link.

(This is on Chrome version 51 - not tried it with other browsers)
